app/models/concerns/itemizable.rb

```diff
@@ -102,14 +102,21 @@ def total_quantity
     line_items.total
   end
 
-  def to_a
+  def line_item_hashes
```

Comment:
This name is technically correct but something about it bugs me. Truth is I'm not entirely sure why we pass these around as hashes in the first place :shrug: but changing is probably not worth it. Maybe `line_item_values`?

Reply:
`hash_items` maybe? to go with `line_items`? I don't actually understand what **line** refers to.

Reply:
Chat GPT is no use
Certainly! Choosing good method names is important for code readability. For a method that returns line items as hashes, you might consider a name that reflects the transformation into hashes. Here are a few suggestions:

    line_items_to_hashes
    convert_line_items_to_hashes
    line_items_as_hashes
    hashify_line_items
    line_items_hash_representation

Choose a name that aligns well with your coding style and accurately conveys the purpose of the method.

Reply:
A "line item" is a real-world term for essentially a row on an invoice or order form.

I think my issue with "hashes" is that it specifies the _type_ it's returning rather than the _meaning_ of the thing that's being returned.

Reply:
`line_item_values` it is. 

The name does make sense, if `line_items` is an array of model objects essentially then `line_item_values` shows that you aren't passing the item itself but just the contents.